examples/GainPlugin/PluginProcessor.cpp

```diff
@@ -28,158 +28,170 @@ AudioProcessorValueTreeState::ParameterLayout createParameterLayout()
                 return String(Decibels::gainToDecibels(value), 1) + "dB";
             },
             nullptr
-        )
+        ),
+        std::make_unique<AudioParameterBool>(
+            "MainMute",
+            "Mute",
+            false
+       )
     );
 
     return params;
 }
-
-//==============================================================================
-GainPluginAudioProcessor::GainPluginAudioProcessor()
+
+//==============================================================================
+GainPluginAudioProcessor::GainPluginAudioProcessor()
      : AudioProcessor (BusesProperties()
                        .withInput  ("Input",  AudioChannelSet::stereo(), true)
                        .withOutput ("Output", AudioChannelSet::stereo(), true)),
-       params(*this, nullptr, JucePlugin_Name, createParameterLayout())
-{
-}
-
-GainPluginAudioProcessor::~GainPluginAudioProcessor()

```

Comment:
This file seemed to be the only one lurking with dos/windows line endings and my editor has converted it. Let me know if that's an issue. Everything else seems to use unix line endings.

Reply:
Yeeaaaa :) nice, thanks, definitely prefer unix line endings